index.html

```diff
@@ -2,162 +2,28 @@
-          <td>Groups the header content in a table</td>
-        </tr>
-        <tr>
-          <td>&lt;tbody&gt;</td>
-          <td>Groups the body content in a table</td>
-        </tr>
-        <tr>
-          <td>&lt;tfoot&gt;</td>
-          <td>Groups the footer content in a table</td>
-        </tr>
-      </tbody>
-    </table>
-    <form action="" method="get">
-      <fieldset>
-        <legend>My first form</legend>
-        <p>
-          <label for="name">Name</label><br />
-          <input
-            required
-            id="name"
-            type="text"
-            name="name"
-            placeholder="Name"
-          />
-        </p>
-        <p>
-          <label for="email">Email</label><br />
-          <input required id="email" type="email" name="email" />
-        </p>
-        <p>
-          <label for="password">Password</label><br />
-          <input required id="password" type="password" name="password" />
-        </p>
-        <p>
-          <label for="level">Your programming level</label><br />
-          <input
-            id="level"
-            type="range"
-            name="level"
-            min="0"
-            max="10"
-            value="1"
-          />
-        </p>
-        <p>
-          What does HTML stand for?<br />
-          <input id="html1" type="checkbox" name="html" value="1" />
-          <label for="html1">Hyperlinks and Text Markup Language</label><br />
-          <input id="html2" type="checkbox" name="html" value="2" />
-          <label for="html2">Home Tool Markup Language</label><br />
-          <input id="html3" type="checkbox" name="html" value="3" />
-          <label for="html3">Hyper Text Markup Language</label><br />
-        </p>
-        <p>
-          What does HTTP stand for?<br />
-          <input id="http1" type="radio" name="http" value="1" />
-          <label for="http1">Hyper Text Translate Protocol</label><br />
-          <input id="http2" type="radio" name="http" value="2" />
-          <label for="http2">Hyper Text Transmission Protocol</label><br />
-          <input id="http3" type="radio" name="http" value="3" />
-          <label for="http3">Hyper Text Transfer Protocol</label><br />
-        </p>
-        <p>
-          <label for="comment">Comment</label><br />
-          <textarea id="comment" name="comment" maxlength="500">
-Comment</textarea
-          >
-        </p>
-        <p>
-          <label for="select">What does HTML stand for?</label><br />
-          <select id="select" name="select">
-            <option disabled selected>Your answer</option>
-            <option value="1">I already answered this question</option>
-            <option value="2">Umm..argh</option>
-            <option value="3">Hyper Text Markup Language</option>
-          </select>
-        </p>
-        <p>
-          <button>Send</button>
-        </p>
-      </fieldset>
-    </form>
-    <hr />
+    <h1 class="text-center">What do we offer</h1>
```

Comment:
Very Cool homework!  But I think it would be better to put h2, because h1 is desirable to use 1 time on the site, because. it shows the main name of the application. Here we have (if you look at the figure) just a small section of the site